testsmocks/activeinactive.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * UpdateHub
+ * Copyright (C) 2017
+ * O.S. Systems Sofware LTDA: contato@ossystems.com.br
+ *
+ * SPDX-License-Identifier:     GPL-2.0
+ */
+
+package testsmocks
```

Comment:
Move  this to a specific package so we don't add more mocks for later conversion.